examples/src/main/scala/caliban/akkahttp/ExampleApp.scala

```diff
@@ -59,13 +70,16 @@ object ExampleApp extends App with GenericSchema[Console with Clock] {
    * "query": "query { characters { name }}"
    * }'
    */
-  val route =
+  val route: Route =
     path("api" / "graphql") {
-      AkkaHttpAdapter.makeHttpServiceM(interpreter)
+      apiRoute // do not call makeHttpServiceM here, or it will be recreated at every call
+    } ~ path("ws" / "graphql") {
+      wsApiRoute
     } ~ path("graphiql") {
       getFromResource("graphiql.html")
     }
 
+  // convention is normally 8080. Keep at 8088?
```

Comment:
Because so many services use 8080, I always end up with conflicts. I'd prefer to keep 8088 for this reason.